lib/inspec/base_cli.rb

```diff
@@ -24,6 +25,32 @@ class << self
       attr_accessor :inspec_cli_command
     end
 
+    def self.start(given_args = ARGV, config = {})
+      check_license! if config[:enforce_license] || config[:enforce_license].nil?
+
+      super(given_args, config)
+    end
+
+    # EULA acceptance
+    def self.check_license!
+      allowed_commands = ['-h', '--help', 'help', '-v', '--version', 'version']
+
+      require 'license_acceptance/acceptor'
+      begin
+        if (allowed_commands & ARGV.map(&:downcase)).empty? &&  # Did they use a non-exempt command?
```

Comment:
Might just be my brain (and it often is!), but the inverse of this makes more sense to me. Something like:

```ruby
!(ARGV.map(&:downcase) - allowed_commands).empty? # man I wish English had a proper antonym for empty.
```

Reply:
I did not to change the existing code, I just moved it without major changes. However I'm open for such changed/optimizations if needed

Reply:
Not needed. Just a suggestion. I'll get to it sooner or later.